.github/workflows/airflow-plugin.yml

```diff
@@ -57,15 +57,16 @@ jobs:
         if: always()
         run: source metadata-ingestion-modules/airflow-plugin/venv/bin/activate && pip freeze
       - uses: actions/upload-artifact@v3
-        if: ${{ always() && matrix.python-version == '3.10' && matrix.extraPythonRequirement == 'apache-airflow>2.6.0' }}
+        if: always()
```

Comment:
I think this one shouldve stayed as-is

Reply:
won't that upload test results for all scenarios then ? everytime with same name ?

One problem I've with adding airflow version in artifact name is that.. sign > is not allowed in artifact name.

Reply:
I'm not aware where these test results show up and how they are used in turn.

Reply:
ah by as-is, I mean `if: ${{ always() && matrix.python-version == '3.10' && matrix.extraPythonRequirement == 'apache-airflow>2.6.0' }}`

the artifact names can remain the default and generic - doesn't need to be specific

Reply:
they're used to produce the `Unit Test Results (<name>)` lines in the CI - basically to combine the results across a bunch of different workflow

Reply:
Could you please add suggestions on the lines ?

```
  - uses: actions/upload-artifact@v3
        if: ${{ always() && matrix.python-version == '3.10' && matrix.extraPythonRequirement == 'apache-airflow>2.6.0' }}
        with:
          name: Test Results (Airflow Plugin ${{ matrix.python-version}})
```

Reply:
ah seems like I was looking at the revision before this commit went in [`b38868f` (#8688)](https://github.com/datahub-project/datahub/pull/8688/commits/b38868f4b502c5dbe55638ce2f5526cd850ab997)

it looks fine now

Reply:
> they're used to produce the Unit Test Results (<name>) lines in the CI - basically to combine the results across a bunch of different workflow

Interesting, I don't see Unit Test Results (Airflow Plugin) in recent workflows executions yet.

Reply:
yeah I think it only kicks in once this is merged to master because it uses a `pull request target` trigger instead of `pull_request`